lib/controller.js

```diff
@@ -170,6 +177,134 @@ Tessel.get = function(opts) {
     }
   });
 };
+/*
+Because of sync problem with master trunk and parallel development on Tessel.list 
+it was necessary to go my own way...
```

Comment:
`Tessel.get` essentially scans for all Tessels (just like `Tessel.list`) and then runs the found Tessels through a bunch of heuristics to determine which one the user likely wants. It can be considered a black box where you always get the Tessel the user would like to operate on.